### ISSUE TITLE
Ability to create dialog from a panel.

### ISSUE THREAD

**Issue by dbauszus-glx** [GEOLYTIX/xyz#1582]
A UI utility method could be tested to turn location views, layer panel, or even layer views into dialogs and returning the view into the placeholder when closed.

**Comment by cityremade** [comment]
Are we talking some top right corner arrow to show the content in a modal? 
That would pop it out of the locations container and add again when modal would be closed?
Or dragging element out of locations ?
I've seen something like this but I can't remember where 🤔

**Comment by dbauszus-glx** [comment]
Yes an optional button for the drawer elements which effectively turns a drawer into a dialog and back into the drawer when closed.

**Comment by cityremade** [comment]
https://jsfiddle.net/cityremade/7s35jx1c/

**Comment by simon-leech** [comment]
> https://jsfiddle.net/cityremade/7s35jx1c/

This is cool - I like it! Can see it being useful for popping out `infoj`, `styles` and `filters` 👍

**Comment by cityremade** [comment]
how and if we handle multiple of these?
